flag.go

```diff
@@ -232,8 +239,14 @@ func (f *FlagSet) SetOutput(output io.Writer) {
 // VisitAll visits the flags in lexicographical order, calling fn for each.
 // It visits all flags, even those not set.
 func (f *FlagSet) VisitAll(fn func(*Flag)) {
```

Comment:
Why only `VistAll` and not `Visit` ?  Should we really change the behavior of `sortFlags()`